allensdk/brain_observatory/behavior/trials_processing.py

```diff
@@ -404,41 +387,149 @@ def get_trial_image_names(trial, stimuli) -> Dict[str, str]:
     }
 
 
-def get_trials(data, licks_df, rewards_df, stimulus_presentations_df, rebase):
+def get_trial_bounds(trial_log: List) -> List:
+    """
+    Adjust trial boundaries from a trial_log so that there is no dead time
+    between trials.
+
+    Parameters
+    ----------
+    trial_log: list
+        The trial_log read in from the well known behavior stimulus pickle file
+
+    Returns
+    -------
+    list
+        Each element in the list is a tuple of the form (start_frame, end_frame)
+        so that the ith element of the list gives the start and end frames of
+        the ith trial. The endframe of the last trial will be -1, indicating that
+        it should map to the last timestamp in the session
+    """
+    start_frames = []
+
+    for trial in trial_log:
```

Comment:
fwiw, i find this nested logic structure a bit hard to grok.
I would have thought something like:
```
start_frames = [event[-1]
                         for trial in trial_log
                         for event in trial['events'] if event[0] == 'trials_start']
<logic that there is >= 1 per trial>
<logic that they are ascending>
```
no change requested